src/style.css

```
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: "Avenir Book", sans-serif;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf), format("ttf");
+}
+
+@font-face {
+  font-family: "Avenir Heavy", sans-serif;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf), format("ttf");
+}
```

```suggestion
@font-face {
  font-family: "Avenir";
  font-weight: 400;
  src: url(fonts/Avenir-Book.ttf), format("ttf");
}

@font-face {
  font-family: "Avenir";
  font-weight: 900;
  src: url(fonts/Avenir-Heavy.ttf), format("ttf");
}
```

You may use just one font-family but use it with different font-weights